src/pages/VeilederInput/VeilederInput.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react'
+import { RouterProvider, createBrowserRouter } from 'react-router-dom'
+
+import {
+  Alert,
+  BodyLong,
+  Button,
+  HStack,
+  Heading,
+  InternalHeader,
+  Spacer,
+  TextField,
+  VStack,
+} from '@navikt/ds-react'
+
+import { Card } from '@/components/common/Card'
+import { FrameComponent } from '@/components/common/PageFramework/FrameComponent'
+import BorgerInformasjon from '@/components/veileder/BorgerInformasjon'
+import { BASE_PATH } from '@/router/constants'
+import { routes } from '@/router/routes'
+import {
+  apiSlice,
+  useGetAnsattIdQuery,
+  useGetPersonQuery,
+} from '@/state/api/apiSlice'
+import { useAppDispatch, useAppSelector } from '@/state/hooks'
+import { veilederBorgerFnrSelector } from '@/state/userInput/selectors'
+import { userInputActions } from '@/state/userInput/userInputReducer'
+
+import { VeilederInputRequestError } from './VeilederInputRequestError'
+
+import styles from './VeilederInput.module.scss'
+
+const router = createBrowserRouter(routes, {
+  basename: `${BASE_PATH}/veileder`,
+})
+
+export const VeilederInput = () => {
+  const dispatch = useAppDispatch()
+  const veilederBorgerFnr = useAppSelector(veilederBorgerFnrSelector)
+
+  const { data: ansatt } = useGetAnsattIdQuery()
+
+  const {
+    isSuccess: personSuccess,
+    isFetching: personLoading,
+    error: personError,
+  } = useGetPersonQuery(undefined, {
+    skip: !veilederBorgerFnr,
+  })
+
+  const hasTimedOut = React.useMemo(() => {
+    const queryParams = new URLSearchParams(window.location.search)
+    return queryParams.has('timeout')
+  }, [])
+
+  const onTitleClick = () => {
+    window.location.href = `${BASE_PATH}/veileder`
+  }
+
+  // Redirect etter 1 time
+  React.useEffect(() => {
+    if (hasTimedOut) return
+    const timer = setTimeout(
+      () => {
+        console.log('Redirecting to timeout')
+        window.location.href = `${BASE_PATH}/veileder?timeout`
+      },
+      60 * 60 * 1000
+    )
+    return () => clearTimeout(timer)
+  }, [])
+
+  const onSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
+    const nyFnr = event.currentTarget.veilderBorgerFnr.value
+    dispatch(userInputActions.setVeilederBorgerFnr(nyFnr))
+    dispatch(apiSlice.util.invalidateTags(['Person']))
+  }
+
```

Comment:
Bør vi ha en enkel form for feilhåndtering her? F.eks en enkel alertboks dersom kall til /person feiler slik at veileder ikke blir på siden med en ufungerende logg inn knapp

Reply:
Se vekk fra denne kommentaren. Jeg så VeilderInputRequestError bare etterpå :)